### ISSUE TITLE
New Configuration: Only fetch blob containers explicitly added

### ISSUE THREAD

**Issue by BlitzkriegSoftware** [microsoft/AzureStorageExplorer#5541]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Problem

When I load the utility, the list of 100s of blob containers load in the UI

### Desired Solution

Add a setting, to not auto-fetch all of the containers (just the ones I just added)

### Alternatives and Workarounds

n/a

### Additional Context

This configuration would be useful for large enterprises with many/many storage accounts, so that we have the option to just connect to ones we explicitly add. E.g. behave more like SSMS where only the SQL accounts we explicitly have connected to are shown. Making this a setting would preserve the backwards compatibility, while allowing enterprise users to have a better experience.

**Comment by MRayermannMSFT** [comment]
@BlitzkriegSoftware you can individually connect to blob containers via the connect dialog (plug icon on the left). That sounds to be what you are looking for. Please give it a try and let us know. :)

**Comment by BlitzkriegSoftware** [comment]
Yes, that works I can add the ones I want explicitly, the PROBLEM is, that the utility tries to iterate through the storage accounts in my subscriptions and a) that takes a while, b) most are not relevant to what I am trying to do in the moment.
![Azure-Storage-Explorer-v1 23 0_TooManyFiles](https://user-images.githubusercontent.com/6732665/160691584-cf045b01-5c0f-4d93-95fc-a00ec07cbdb1.png)
![Azure-Storage-Explorer-v1 23 0_Version](https://user-images.githubusercontent.com/6732665/160691601-8ddcad45-5cf4-49d5-8b9b-91ad06ea0deb.png)

**Comment by MRayermannMSFT** [comment]
@BlitzkriegSoftware if you go to the account panel (person icon) you can un-check all of those subscriptions, and storage explorer won't try to access resources from them. Also, if you are attaching containers directly and you are not authenticating with Azure AD (so if you are using key or SAS) then you could even go as far as completely signing out of Storage Explorer.

**Comment by MRayermannMSFT** [comment]
Closing due to lack of reply. If you are still running into this problem we suggest you open an Azure support ticket from the Azure portal. Alternatively you can open a new issue here.
